packages/preview/paddling-tongji-thesis/0.1.1/init-files/sections/01_intro.typ

```diff
@@ -0,0 +1,217 @@
+#import "../../paddling-tongji-thesis/elements.typ": *
```

Comment:
This import should be replaced with a `@preview/...` import too (same for all the other files in this directory).